README.md

```diff
@@ -20,142 +20,96 @@ This section documents how developers can build and run ArkCase.  (For non-devel
-
-### Run the Configuration Server
-
-Starting with version 3.3.1, ArkCase requires a separate configuration server, based on Spring Cloud Config Server (more info here: https://spring.io/projects/spring-cloud-config).  To start the config server, take these steps:
-
-* Download the most recent config-server.jar file from here: https://github.com/ArkCase/acm-config-server/releases
-* Start the server process with this command: `java -jar config-server-0.0.1.jar`, replacing `0.0.1` with the version you downloaded.
-
-The config server runs on port 9999 by default.  To run on a different port, add the server.port option to the command, like so: `java -Dserver.port=8888 -jar config-server-0.0.1.jar`, replacing `8888` with the desired port.
-
-### Configure Tomcat
-
-#### Tomcat Native Connector
-
-Make sure the Tomcat native connector library is being used.  
-
-* MacOS: open terminal, issue the command `brew install tomcat-native`, and follow any directions you see at the end... Of course you must already have `brew`; see <https://brew.sh/> if you don't already have it.
-* Windows: download from https://tomcat.apache.org/download-native.cgi. 
-* Linux: Information on building for Linux is available from the same URL (https://tomcat.apache.org/download-native.cgi).
-
-#### Tomcat TLS Configuration
-
-In your Tomcat 9 installation, edit the `conf/server.xml` file, and add the following connector, below the existing connector for port 8080:
-
-```xml
-    <Connector port="8843"
-           maxThreads="150" SSLEnabled="true" secure="true" scheme="https"
-           maxHttpHeaderSize="32768"
-           connectionTimeout="40000"
-           useBodyEncodingForURI="true"
-           address="0.0.0.0">
-      <UpgradeProtocol className="org.apache.coyote.http2.Http2Protocol" />
-      <SSLHostConfig protocols="TLSv1.2" certificateVerification="none">
-        <Certificate certificateFile="${user.home}/.arkcase/acm/private/acm-arkcase.crt"
-                    certificateKeyFile="${user.home}/.arkcase/acm/private/acm-arkcase.rsa.pem"
-                    certificateChainFile="${user.home}/.arkcase/acm/private/arkcase-ca.crt"
-                    type="RSA" />
-      </SSLHostConfig>
-    </Connector>
+- [Balena Etcher](https://balena.io/etcher) for creating bootable drives.
+- [Helm 3.12.1](https://helm.sh/docs/intro/install/) for managing Kubernetes applications.
+- A functioning [Kubernetes cluster](https://kubernetes.io/docs/setup/).
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for interacting with your Kubernetes cluster.
+- [Docker](https://docs.docker.com/get-docker/) for containerization.
+
+### Setting Up ubuntu on your Laptop
+
+**Note:** If you already have Ubuntu installed or using an Ubuntu laptop, you can skip this step.
+
+### Install Ubuntu
+
```

Comment:
Point out again here the highly destructive nature of this step.

Also say that if they take this path, they should be sure to select Ubuntu 22.04 LTS, and not any other version that is available at the Ubuntu download site.